pkg/operator/ceph/config/keyring/store.go

```diff
@@ -34,6 +34,9 @@ var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-cfg-keyring")
 
 const (
 	keyringFileName = "keyring"
+
+	// KeyringAnnotation identifies a Kubernetes Secret as a cephx keyring file
+	KeyringAnnotation = "rook-cephx-keyring"
```

Comment:
maybe we don't need the rook prefix?
```suggestion
	KeyringAnnotation = "cephx-keyring"
```